centos-7.template

```diff
@@ -166,6 +168,18 @@ EOF
 
 systemctl enable docker
 
+# Show a warning banner when using yum to install software
+mv /usr/bin/yum /usr/bin/yum-unsupported
+# Place holder for base64 encode yum-wrapper script
+cat > yum-wrapper.base64 << EOF
```

Comment:
less fragile than including the script and using escaping... but I can try to see if this suggestion works as expected, as it is not really a bash substitution that happens AFAIR.